src/components/BounceActivityContainer/ActivityFilter/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { Button } from "@sendgrid/ui-components/button";
+import { TextInput } from "@sendgrid/ui-components/text-input";
+import { Select } from "@sendgrid/ui-components/select";
+import { Row } from "../../Row";
+import { Column } from "../../Column";
+import "./index.scss";
+
+const ActivityFilter = ({
+  searchToken,
+  updateSearchToken,
+  updateSearchCategory,
+  filterOptions,
+  addFilter,
+  removeFilter,
+  isValidFilter,
+}) => (
+  <div className="filter-wrap">
+    <div className="filter-header">
+      <p className="filter-title">
+        {"Search activity by "}
+        <a className="filter-option-toggle" href="#">
+          all
+        </a>
+        {" of the following:"}
+      </p>
+      <div className="btn-list">
+        <Button type="secondary" small>
+          Clear Search
+        </Button>
+      </div>
+    </div>
+    <div className="filter-list">
+      <Row>
+        <Column width={3} offset={1}>
+          <Select
+            className="select-filter"
+            onChange={updateSearchCategory}
+            name="primary-filter"
+            defaultValue={{ label: "Bounce Action", value: "Bounce Action" }}
```

Comment:
let's store these in variables like const defaultFilterOption = { label: "Bounce Action", value: "Bounce Action" } and const filterOptions = [defaultFilterOption, {...}, {...}] to make the JSX more terse